janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -1810,7 +1811,22 @@ private void testCollection(Cardinality cardinality, String property, Object val
             assertEquals(v1, getOnlyElement(graph.query().has(property, Geo.WITHIN, Geoshape.circle(2.0, 2.0, 0.1)).vertices()));
         }
 
-
+        
+        // Test traversal property drop Issue #408
+        GraphTraversalSource g = graph.traversal();
+        g.V().drop().iterate();
+        clopen();//Flush the index
+        g = graph.traversal();
+        v1 = g.addV().property(property, value1).property(property, value2).next();
+        Vertex v2 = g.addV().property(property, value1).property(property, value2).next();
+        clopen();//Flush the index
+        g = graph.traversal();
+        assertEquals(2, g.V().has(property, value1).toList().size());
+        g.V().properties().drop().iterate();
+        clopen();//Flush the index
+        g = graph.traversal();
+        assertEquals(false, g.V().has(property, value1).hasNext());
```

Comment:
`assertEquals(false, ...)` == `assertFalse(...)` but the latter is shorter and easier to read.

Similarly below.